src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateArray = [];
+  let stateCopy = { ...state };
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      stateCopy = {
+        ...stateCopy,
+        ...action.extraData,
+      };
+
+      stateArray.push({ ...stateCopy });
```

Comment:
dont repeat it in every case, move it after switch case

Reply:
<img width="773" alt="image" src="https://github.com/mate-academy/js_stateful-clones/assets/70220667/08425eb3-db5d-47f8-ad31-4d7b904f3bf1">
still not fixed